deps/rabbitmq_amqp1_0/src/rabbit_amqp1_0_incoming_link.erl

```diff
@@ -226,3 +225,47 @@ incoming_flow(#incoming_link{ delivery_count = Count }, Handle) ->
     #'v1_0.flow'{handle         = Handle,
                  delivery_count = {uint, Count},
                  link_credit    = {uint, ?INCOMING_CREDIT}}.
+
+max_incoming_credit() -> 
+    ?INCOMING_CREDIT.
+
+-spec adjust_credits(Method :: ack | transfer, 
```

Comment:
instead of `ack` I'd suggest something like `queue_settled` or similar - the ack terminology is just an incident of AMQP legacy extensions.

Reply:
I used `settled` as it is in `rabbit_channel`.